pkg/datadog/config/config.go

```diff
@@ -133,6 +133,10 @@ func (c *Config) Validate() error {
 		return err
 	}
 
+	if c.HostMetadata.ReporterPeriod <= 0 {
+		return errors.New("reporter_period must be a positive duration")
```

Comment:
We would like to prevent users from accidentally spamming the backend by setting the reporter period too low. The Agent currently has a minimum of 5 minutes, so unless you believe there to be strong use cases for even lower values, I think we should do the same here.
```suggestion
	if c.HostMetadata.ReporterPeriod < 5 * time.Minute {
		return errors.New("reporter_period must be 5 minutes or higher")
```

Reply:
Yes, that makes sense, we can go with minimum 5 minutes. It's already very good like this. Thank you!